lib/modules/subredditManager.js

```diff
@@ -777,6 +782,8 @@ function redrawSubredditBar() {
 				if (module.options.linkMod.value) shortCutsHTML += `<span class="separator">-</span><a class="subbarlink ${specialButtonSelected.mod}" href="/r/mod/">Mod</a>`;
 				if (module.options.linkModqueue.value) shortCutsHTML += '<span class="separator">-</span><a class="subbarlink" href="/r/mod/about/modqueue">Modqueue</a>';
 			}
+
+			if (module.options.linkSaved.value) shortCutsHTML += '<span class="separator">-</span><a class="subbarlink" href="/user/me/saved">Saved</a>';
```

Comment:
To avoid adding clutter, I think this should only show when the user is logged in, i.e. `if (loggedInUser())`. On the other hand, many people with accounts browse while logged out, and they'll be prompted to log in after clicking this shortcut. 

Bonus points: add an option like `linkSavedLoggedOut`? What do you think, and are you interested in implementing it?